velox/functions/prestosql/tests/ArraySortTest.cpp

```diff
@@ -229,7 +229,8 @@ class ArraySortTest : public FunctionBaseTest,
         test<StringView>();
         break;
       case TypeKind::MAP:
-        test<TestMapType>();
+        VELOX_ASSERT_THROW(
```

Comment:
Let's remove all test code related to maps and just have a dedicate test to verify that sorting arrays of maps is not allowed.